framework/PageDashboard/PageDashboardChart.tsx

```diff
@@ -83,9 +85,20 @@ export function PageDashboardChart(props: {
           }
           containerComponent={
             <ChartVoronoiContainer
-              labels={(point: { datum: { x: string | number; y: string | number } }) => {
+              labels={(point: {
+                datum: {
+                  x: string | number;
+                  y: string | number;
+                  hosts_added?: string | number;
```

Comment:
These chart usage specific value should not be in the generic PageDashboardChart.

Maybe make this a tooltip details?
```
tooltipDetails: string[] 
```

Then we could render those under the item in the tooltip.

Note: The tooltips were enhanced in this [PR](https://github.com/ansible/ansible-ui/pull/811) to use the group label in the tooltip.